### PR TITLE
Add recipes to sidebar 

### LINKED ISSUE #35: Add Saved Recipes To Sidebar
**Is your feature request related to a problem? Please describe.**
Users need to be able to save recipes to the sidebar.

**Describe the solution you'd like**
Users save selected recipes to the sidebar to have access/view all saved recipes.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Created hook to store recipes and display them in sidebar.
2. Move BlobsContainer styled-component out of `MainContainer` and gave it props and interface template.
3. Modified `topIngredients`.
4. Deleted `Blobs.tsx`.

## Purpose
Add recipes to sidebar and fix rerender issue.

## Approach
React hooks and React typescript.

Closes #35

### CODE REVIEW

**Comment by kmausolf** [comment]
Hmmm.  I've been noticing that package-lock gets deleted before every PR.  Is this intentional?

**Comment by kmausolf** [comment]
Oh, maybe it's just changing significantly.  Make sure all team members are on the same node version!

File: src/components/MainContainer.tsx [hunk 1]
```diff
@@ -25,6 +29,16 @@ const Container = styled.div`
   grid-template-columns: 350px 1fr;
 `;
 
+const BlobsContainer = styled.div<BlobsContainerProps>`
+grid-area: 1 / 2 / 2 / 3;
+background-color: #ebe6da;
+padding: 10px;
+display: grid;
+grid-template-columns: ${props => !props.userSearchedRecipes ? `1fr 1fr 1fr 1fr` : `1fr 1fr`};
+grid-auto-rows: 250px;
```
Comment (krferrell) [inline src/components/MainContainer.tsx:38]: Nice use of those props on the styled components!

File: src/components/MainContainer.tsx [hunk 2]
```diff
@@ -70,14 +71,22 @@ const MainContainer = () => {
     return instructions;
   };
 
+  const addIngredient = (ingredient: string) => {
```
Comment (krferrell) [inline src/components/MainContainer.tsx:74]: should this function have a return type of void since it is not returning anything?

**Review by Lap343 — APPROVED** [review]
Looking good

**Review by dannysanchez559 — APPROVED** [review]
nice use of your guys hooks and spread operators

**Review by krferrell — APPROVED** [review]
Coming along nicely!

**Pull request merged**